packages/eslint-config-catalyst/react.js

```diff
@@ -6,6 +6,7 @@ const config = {
   rules: {
     'react/react-in-jsx-scope': 'off',
     'react/prefer-read-only-props': 'off',
+    'react/destructuring-assignment': 'off',
```

Comment:
With the new changes to Next 15 you need to await things like `searchParams` and `params`. Destructuring props makes this a bit annoying in some scenarios.